File: PST/FUEL/src/FUEL/SysEventLog.vb
Imports System
Imports System.Diagnostics

Namespace FUEL
    Public Class SysEventLog
        ' Methods
        Public Shared Sub WriteLog(ByVal LogName As String, ByVal Msg As String, ByVal EntryType As EventLogEntryType)
            If Not EventLog.SourceExists(LogName) Then
                EventLog.CreateEventSource(LogName, LogName)
            End If
            EventLog.WriteEntry(LogName, Msg, EntryType)
        End Sub

    End Class
End Namespace

File: PST/FUEL Source Code/FUEL/Backup/SysEventLog.cs
// Decompiled with JetBrains decompiler
// Type: FUEL.SysEventLog
// Assembly: FUEL, Version=2.4.8.0, Culture=neutral, PublicKeyToken=null
// MVID: A3D630D0-0620-425D-8CFB-B67B2AC75507
// Assembly location: C:\hp\FUEL\FUEL.dll

using System.Diagnostics;

namespace FUEL
{
  public class SysEventLog
  {
    [DebuggerNonUserCode]
    public SysEventLog()
    {
    }

    public static void WriteLog(string LogName, string Msg, EventLogEntryType EntryType)
    {
      if (!EventLog.SourceExists(LogName))
        EventLog.CreateEventSource(LogName, LogName);
      EventLog.WriteEntry(LogName, Msg, EntryType);
    }
  }
}
